thirdparties/etcdclient/Makefile

```diff
@@ -27,7 +38,7 @@ intall-go:
 
 install-etcdclient:
 	mkdir -p $(pwd)/tmp/gosrc/src/go.etcd.io
-	cd $(pwd)/tmp/gosrc/src/go.etcd.io && git clone --branch v3.4.0 https://gitee.com/mirrors/etcd
+	cd $(pwd)/tmp/gosrc/src/go.etcd.io && git clone --branch v3.4.0 --depth=1 https://gitee.com/mirrors/etcd
```

Comment:
It will only clone the most recent commit. Since we will certainly not use the commit history when we are building etcd, ignoring the commit history will make it faster to clone.